src/pages/CertificatesViewAll.js

```diff
@@ -20,8 +28,56 @@ const headersMap = {
     'Carbon Embodiment',
     'Status',
   ],
-  emma: ['Date', 'Commitment'],
-  reginald: [],
+  emma: [
+    'Date',
+    'Electric energy use',
+    'Production Start',
+    'Production End',
+    'Status',
+  ],
+  reginald: [
```

Comment:
What about Connor ? ( maybe you can add something because when I switch to Connor i get an error )

EDIT: I have to double check that ..

Reply:
The story does not mention conor - outside of scope